src/PowerSystems.jl

```diff
@@ -371,6 +371,8 @@ import InfrastructureSystems:
     get_time_series_names,
     get_next_time_series_array!,
     get_next_time,
+    get_units_info,
+    set_units_info!,
```

Comment:
I'm not sure that we should export these. What are you thinking here?

Reply:
well my intent wasn't to export them but just use them in PSY, maybe I should just use `IS. get_units_info` in the code if this approach exposes these function to the user.

Reply:
Oh sorry, I misread. For some reason I thought these were exported in the other PR. Please ignore.